src/subclass/application.rs

```diff
@@ -0,0 +1,529 @@
+// Copyright 2019, The Gtk-rs Project Developers.
+// See the COPYRIGHT file at the top-level directory of this distribution.
+// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
+use gio_sys;
```

Comment:
You have the right to add an empty line before this one. :)

Reply:
done